api/src/accounts.rs

```diff
@@ -124,6 +127,48 @@ impl AccountsApi {
         .await
     }
 
+    /// Get account resources
+    ///
+    /// Retrieves all account resources for a given account and a specific ledger version.  If the
+    /// ledger version is not specified in the request, the latest ledger version is used.
+    ///
+    /// The Aptos nodes prune account state history, via a configurable time window.
+    /// If the requested ledger version has been pruned, the server responds with a 410.
+    #[oai(
+        path = "/accounts/:address/balance/:asset_type",
+        method = "get",
+        operation_id = "get_account_balance",
+        tag = "ApiTags::Accounts"
+    )]
+    async fn get_account_balance(
+        &self,
+        accept_type: AcceptType,
+        /// Address of account with or without a `0x` prefix
+        address: Path<Address>,
+        asset_type: Path<AssetType>,
```

Comment:
How do you differentiate between string inputs for asset type?

Right now, asset_type on the path is a string, but then you somehow differentiate when it comes in.